src/styles/components/header.scss

```diff
@@ -0,0 +1,14 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
Consider using a variable for the padding value if it's a common spacing in your project to ensure consistency and ease of maintenance.